lib/views/bottom_navigation_bar/navigation_bottom_bar.dart

```diff
@@ -6,6 +6,7 @@ import "package:proxima/views/bottom_navigation_bar/navigation_bar_routes.dart";
 /// It contains the navigation routes to the different pages.
 class NavigationBottomBar extends HookConsumerWidget {
   static const navigationBottomBarKey = Key("navigationBottomBar");
+  static const selectedIndex = 0;
```

Comment:
This should not be static const as it might change in the future. We should put it as a field of the widget and have it passed as argument to the constructor.

Reply:
Fixed by 52385d5